Open_Foam/Domain.ipynb

```diff
@@ -79,11 +78,48 @@
     "    arc 2 5 (22.6274 22.6274 0)\n",
     "    arc 7 10 (0.3535 0.3535 1)\n",
     "    arc 8 11 (22.6274 22.6274 1)\n",
-    ");</pre>\n",
+    ");\n",
+    "```\n",
+    "\n",
+    "At this stage the `blockMeshDict` file is completely setup for creating the geometry of mesh of half of the CFD domain. User should save the `blockMeshDict` file and compile it to by running the following command in the working directory:\n",
+    "\n",
+    "```C++\n",
+    "blockMesh\n",
+    "```\n",
+    "\n",
+    "The output of compiling the created `blockmesh` file is visualized in Fig. 4.\n",
+    "\n",
+    "<img src=\"./Images/4halfMesh.png\" width=\"500\">\n",
+    "Fig. 4. The half CFD domain geometry and mesh produced by `blockMesh` utility.\n",
+    "\n",
+    "Now it is time to finish the creation of CFD domain's geometry and mesh by using the mirror boolean operation. This step is done by executing `mirrorMesh` utility. `mirrorMesh` would reflect the existing geometry and mesh with respect to a defined plane of symmetry. The settings for this command are defined in the `mirrorMeshDict` file located in constant/polyMesh directory as follows:\n",
+    "\n",
+    "```C++\n",
+    "planeType       pointAndNormal;\n",
+    "\n",
+    "pointAndNormalDict\n",
+    "{\n",
+    "    basePoint       (0 0 0);\n",
+    "    normalVector    (0 -1 0);\n",
+    "}\n",
+    "\n",
+    "planeTolerance  1e-06;\n",
+    "```\n",
     "\n",
```

Comment:
Explain the details of the commands and settings above. What each setting is and how you define it?